cegs_portal/search/templates/search/v1/dna_feature.html

```diff
@@ -241,6 +245,18 @@
     {% for feature, sources, targets in feature_reos %}
         {% if forloop.first %}
             <div style="display: flex; justify-content: space-between">
+                {% if profile_header_feature_type != "gene" and profile_header_feature_type != "transcript" and profile_header_feature_type != "exon" %}
```

Comment:
I think a better way of thinking about this is features with names vs. features without names. If the feature has a name, use the name. If it doesn't, use `[feature_type] at [location]`.

Reply:
more specifically, use whether feature.name is null or not to determine which style to use.